frontend/src/app/fullname.pipe.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { FullnamePipe } from './fullname.pipe';
+import { PersonModel } from './models/person.model';
+
+describe('FullnamePipe', () => {
+
+  const pipe = new FullnamePipe();
+
+  it('should not display anything if no first name or lastname of nickname', () => {
+    const person : PersonModel = {} as PersonModel
+    expect(pipe.transform(person)).toBe('');
+  });
+
+  it('should display first name and lastname when no nick name', () => {
+    const person : PersonModel = {
```

Comment:
I saw you use this in several places: `const myVar: MyType = {} as MyType`. I did not checked but I think the type inference allows `const myVar = {} as MyType` right?